fix/logging: use ISO format for date & time

ustulation opened the pull request:
Logs to web-server will come from different machines situated in different time zones. The previous code sent local machine's time which is not very useful for Web-Servers. This code fixes that by making sure ISO format with UTC offset is used

<!-- Reviewable:start -->

---

This change is [<img src="https://reviewable.io/review_button.svg" height="35" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/maidsafe/maidsafe_utilities/53)

<!-- Reviewable:end -->

ustulation committed:
fix/logging: use ISO format for date & time
Logs to web-server will come from different machines situated in different time zones. The previous code sent local machine's time which is not very useful for Web-Servers. This code fixes that by making sure ISO format with UTC offset is used

maidsafe-highfive commented:
r? @canndrew

(maidsafe_highfive has picked a reviewer for you, use r? to override)

ustulation committed:
fix/logging: send 'id' as string
Webservers written in nodejs are unable to handle 64bit numbers. Hence converting the unique-id to string for them to be able to parse properly.

ustulation commented:
r? @krishnaIndia

krishnaIndia commented:
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/maidsafe/maidsafe_utilities/53)*
<!-- Sent from Reviewable.io -->
